monkey/monkey_island/cc/resources/report_generation_status.py

```diff
@@ -14,7 +15,7 @@ def __init__(self, agent_repository: IAgentRepository):
     def get(self):
```

Comment:
Why is this not protected?

Reply:
Fixed, the decorator was accidently applied to the wrong function the second time this was done.